apps/src/dance/DanceVisualizationColumn.jsx

```diff
@@ -162,6 +143,26 @@ class DanceVisualizationColumn extends React.Component {
 }
 
 const styles = {
+  visualization: {
+    width: MAX_GAME_WIDTH,
+    height: GAME_HEIGHT
+  },
+  container: {
+    touchAction: 'none',
+    background: '#fff',
+    position: 'relative',
+    overflow: 'hidden'
+  },
+  loadingContainer: {
+    // The value of display will be controlled externally by StudioApp.
```

Comment:
nit: "is controlled" instead of "will be controlled" feels slightly more clear (to me at least)